cst438_gradebook/src/test/java/com/cst438/EndToEndTestAssignments.java

```diff
@@ -78,15 +78,15 @@ public void testCreateAssignment() throws Exception {
 			driver.findElement(By.xpath("//input[@name='handleSubmit']")).click();
 			Thread.sleep(SLEEP_DURATION);
 			
-			Optional<Assignment> a = assignmentRepository.findByCourseIdAndName(TEST_COURSE_ID, TEST_ASSIGNMENT_NAME);
+			Optional<Assignment> a = assignmentRepository.findAssignmentByCourseIdAndName(TEST_COURSE_ID, TEST_ASSIGNMENT_NAME);
 			if (a != null) {
 				assignmentRepository.delete(a);
```

Comment:
delete requires a single entity below will be the first item of the list if it's populated
```suggestion
				assignmentRepository.delete(a.get(0));
```